guides/v2.1/cloud/cdn/fastly-waf-service.md

```diff
@@ -6,26 +6,31 @@ functional_areas:
+- **Default WAF policy**—The default WAF policy, configured and maintained by Fastly, provides a collection of security rules tailored to protect your {{ site.data.var.ee }} web applications from a wide range of attacks, including injection attacks, malicious inputs, cross site scripting, data exfiltration, HTTP protocol violations, and other [OWASP Top Ten](https://www.owasp.org/index.php/Top_Ten) security threats.
 - **WAF onboarding and enablement**—Magento works with you to plan and schedule the WAF rollout to deploy the WAF, tune the default WAF policy in your environment, and enable the service.
-- **Operations support**—
+- **Operations and maintenance support**—
   - Magento and Fastly set up and manage your WAF logs and alerts.
   - Customer support tickets related to WAF issues that block legitimate traffic are triaged as Priority 1 issues. 
+  - WAF version upgrades to ensure immediate coverage for new or evolving exploits. See [WAF maintenance and upgrades](#waf-maintenance-and-updates).
+  
+The Managed Cloud WAF is included with both {{ site.data.var.ece }} Pro and Starter subscriptions at no additional cost. The WAF service is designed to protect production environments. It is not available for staging environments.
 
-The Magento WAF service is integrated into Fastly's global nodes and runs as a part of the cache logic within the Fastly CDN service. Magento enables the WAF in your production environment with a default WAF policy based on [ModSecurity Rules from Trustwave SpiderLabs](https://www.trustwave.com/Products/Application-Security/ModSecurity-Rules-and-Support/) and the OWASP Top Ten security threats.
+## How it works
 
+The Managed Cloud WAF is integrated into Fastly's global nodes and runs as a part of the cache logic within the Fastly CDN 
+service. Magento enables the WAF in your production environment with a default WAF policy based on [ModSecurity Rules from Trustwave SpiderLabs](https://www.trustwave.com/Products/Application-Security/ModSecurity-Rules-and-Support/) and the OWASP Top Ten security threats.
 
-Once deployed, the WAF monitors web and admin traffic to identify any suspicious activity. It filters HTTP and HTTPS traffic (GET and POST requests) against the WAF ruleset and blocks traffic that is malicious or does not comply with specific rules. It can block SQL injection attacks, cross-site scripting attacks, and more.
+The Managed Cloud WAF analyzes, filters HTTP and HTTPS traffic (GET and POST requests) against the WAF ruleset, and blocks traffic that is malicious or does not comply with specific rules. Because the WAF service filters only origin-bound traffic that attempts to refresh the cache, most attack traffic is stopped at the Fastly cache, protecting your origin traffic from malicious attacks.  Because the WAF service is integrated with Fastly, it also preserves cache performance—introducing only an estimated 1.5 milliseconds (ms) to 20 ms of latency to every non-cached request.
```

Comment:
The clause with "filters HTTP..." seems to break up the verb flow. Double check...